### ISSUE TITLE
Questions about trainting strategy

### ISSUE THREAD

**Issue by VV-Hope** [miaodd98/ITrans#1]
What are your iteration counts on the CelebA-HQ dataset? Why does my model perform well on the training set but poorly on the test set? Could you please provide your training strategy?

**Comment by miaodd98** [comment]
Actually the mask dataset has a huge influence on the outcome. I trained about 400 epochs on CelebA-HQ with a fixed learning rate from scratch, and changed the mask dataset in the middle. At the beginning training with small mask and then turning to huge mask for better generation performance. Some of the masks are generated in a way like LaMa: Resolution-robust Large Mask Inpainting with Fourier Convolutions, you can check that.

**Comment by VV-Hope** [comment]
Thanks for your answer. Do you use --model 3 directly during training?

**Comment by VV-Hope** [comment]
Thanks for your answer. Do you use --model 3 directly during training?

**Comment by miaodd98** [comment]
Yes, and sometimes use model 1 to train the edge detection network with different masks at the beginning of training.

**Comment by VV-Hope** [comment]
Thanks for your answer. You are so kindly~
